{% extends "layout.html" %}

{% block title %}Engineering Success{% endblock %}
{% block lead %}Demonstrate engineering success in a part of your project by going through at least one iteration of the
engineering design cycle.{% endblock %}

{% block page_content %}

<div class="row mt-4">
  <h1>Designing the peptide</h1>
  <p>The Design–Build–Test–Learn (DBTL) cycle is integral to any synthetic biology-based engineered solution for biological problems. As a result, to arrive at the best possible peptide, we performed multiple iterations of the engineering cycle. Each iteration involved designing our peptide, building it and finally testing it in-silico. The changes involved in each iteration and the information learnt have been extensively documented.</p>
  <img src="https://static.igem.wiki/teams/4200/wiki/engsuccess-dbtlcycle.png" alt="*The design, build test learn cycle graphic">
  <h2>
    The DBTL cycle
    <br>
    <small class="text-muted"><em>Design, Build, Test and Learn</em></small>
  </h2>
  <p>The predicted structures of MAM7 and HPD of Fibronectin were docked with each other on ClusPro [1], followed by visualisation and observation of these interacting residues on Pymol [2]. The sequence signature of the structural repeats in the proteins of Fibronectin and MAM7 were detected using HHrepID[3]—a tool for de novo identification of repeats in protein sequences. This confirmed the existence of 5 continuous repeats of type 1 modules, forming the heparin-binding domains of Fibronectin and the presence of seven mammalian cell entry domains (MCEs) in MAM7. Fibronectin is known to require five continuous repeats to dock with MAM7; as a result, designing a peptide that would bind to only one of these domains was deemed sufficient.[4]

  </p>
  <p>Each of the five repeats of the type I module present in Fibronectin was considered a particular peptide, and each of these peptides was separately docked with MAM7 to test its binding energies using ClusPro [1]. Amber, an energy minimisation tool, was employed for each of these interactions. It was concluded that peptides numbered three and five derived from modules three and five of fibronectin, respectively, docked with the MCE3 of MAM7, giving the highest docking scores. 
    It was seen that peptide five (wild type) had the highest docking score with MCE3. Thus, this peptide was taken as the basis for the next set of modifications.</p>
    <img src="" alt="*Image of peptide 5 docked to MAM7">
    <p>Schrodinger Maestro [5]. was used to visualise and study the specific amino acids involved in the interaction between the peptide and MAM7. The residues within 5 A° were assumed to be close enough for the interaction and thus were considered during the evaluation. Further, this was verified by subjecting the peptide to an alanine scan on BUDE. An alanine scan replaces all the amino acid residues in a peptide with alanine. It compares the values of the respective Gibbs free energies to determine the extent of contribution of every amino acid in the docking process[6]. Five such mutations were recognised in the wild type. This was followed by improving the interactions by substituting corresponding amino acids. These replacements were made based on the study 
      [7], but each of them had a plethora of possibilities. For example, methionine (present at the 29th position) could be replaced by Glutamic acid, Aspartic acid,  Phenylalanine, Tyrosine, Tryptophan, Histidine or Leucine. </p>
      <p>Therefore, the docked structure was subjected to mutation Cutoff Scanning Matrix (mCSM) [8], which helped us analyse which mutations were truly beneficial. However, mCSM does not account for conformational changes that these mutations may induce, so each modified structure was subjected to an alanine scan to verify the substitutions. </p>
      <p>With newly learnt information, the peptide was being subjected to new changes. Hence, it was necessary to predict the structure of the peptide and dock it with MAM7 at every stage. This gave us twelve variants of the peptide with different combinations of the various substitutions that could be made. These were then visualised on Schrodinger Maestro and further analysed using Prodigy to identify the strongly interacting peptides. </p>
      <p>Four of the twelve variants were observed, and the best among these was selected for the next round. The mutations performed in the previous iteration introduced new amino acids in the docked structure, opening up an avenue for a new set of substitutions and hence the process was repeated. Subsequently, a  new set of variants was obtained (16 variants). This was further subjected to docking and analysis to select the next best peptide.  At this stage, to prevent the peptides from mutually interacting and consequently decreasing their efficiency, we performed AGGRESCAN [9] on the four peptides obtained from the last iteration, to highlight the hotspots of aggregation. One such hotspot involving eight amino acids was observed and was rectified by substituting amino acid (C34) with Asparagine in every peptide. After this process, our very own software—PEP_MOD, helped us increase the scope of possible mutations.
      </p>
  <h3>The software</h3>
  <p>Two rounds of running the peptide through the software gave us the final one. The structure of the peptide was then predicted and docked with MAM7, and run through Prodigy [10] to compare their respective binding affinities and Kd values. These values are tabulated below:
  </p>

  *table*

  <h1>References:</h1>
  <ol>
    <li> “ClusPro 2.0: protein-protein docking.” https://cluspro.bu.edu/login.php (accessed Oct. 02, 2022).
      I. T. Desta, K. A. Porter, B. Xia, D. Kozakov, and S. Vajda, “Performance and Its Limits in Rigid Body
      Protein-Protein Docking,” Structure, vol. 28, no. 9, pp. 1071-1081.e3, Sep. 2020, doi: 10.1016/j.str.2020.06.006.
      S. Vajda et al., “New Additions to the ClusPro Server Motivated by CAPRI”, doi: 10.1002/prot.25219.
      D. Kozakov et al., “The ClusPro web server for protein-protein docking”, doi: 10.1038/nprot.2016.169.
      D. Kozakov et al., “How Good is Automated Protein Docking?”, doi: 10.1002/prot.24403.
    </li>
    <li>“PyMOL | pymol.org.” https://pymol.org/2/ (accessed Oct. 02, 2022).
    </li>
    <li>“HHrepID BioCode | BioCodeKB - Learn Bioinformatics.”
      https://www.biocode.ltd/biocodekb-bioinformatics-knowledgeb/HHrepID (accessed Oct. 02, 2022).
    </li>
    <li>A. M. Krachler and K. Orth, “Functional characterization of the interaction between bacterial adhesin
      Multivalent Adhesion Molecule 7 (MAM7) protein and its host cell ligands,” J. Biol. Chem., vol. 286, no. 45, pp.
      38939–38947, Nov. 2011, doi: 10.1074/JBC.M111.291377.
    </li>
    <li>“Maestro | Schrödinger.” https://www.schrodinger.com/products/maestro (accessed Oct. 02, 2022).
    </li>
    <li>A. A. Ibarra et al., “Predicting and Experimentally Validating Hot-Spot Residues at Protein-Protein
      Interfaces,” ACS Chem. Biol., vol. 14, no. 10, pp. 2252–2263, Oct. 2019, doi:
      10.1021/ACSCHEMBIO.9B00560/SUPPL_FILE/CB9B00560_SI_002.XLSX.
    </li>
    <li>N. Z. Xie, Q. S. Du, J. X. Li, and R. B. Huang, “Exploring Strong Interactions in Proteins with Quantum
      Chemistry and Examples of Their Applications in Drug Design,” PLoS One, vol. 10, no. 9, p. 137113, Sep. 2015, doi:
      10.1371/JOURNAL.PONE.0137113.
    </li>
    <li>D. E. V. Pires, D. B. Ascher, and T. L. Blundell, “mCSM: predicting the effects of mutations in proteins using
      graph-based signatures,” Bioinformatics, vol. 30, no. 3, pp. 335–342, Feb. 2014, doi:
      10.1093/BIOINFORMATICS/BTT691.
    </li>
    <li>“Aggrescan: Prediction of ‘hot spots’ of aggregation in polypeptides.” http://bioinf.uab.es/aggrescan/
      (accessed Oct. 02, 2022).
    </li>
    <li>“Prodigy Webserver.” https://wenmr.science.uu.nl/prodigy/ (accessed Oct. 02, 2022).</li>
  </ol>
</div>

{% endblock %}
