{% extends "layout.html" %}

{% block title %}Engineering Success{% endblock %}

{% block page_content %}

<div class="row mt-4">

  <h1 class="reveal-on-scroll">Designing the peptide</h1>

  <p class="reveal-on-scroll"><b>The Design–Build–Test–Learn (DBTL) cycle</b> is integral to any synthetic biology-based
    engineered solution for biological problems. As a result, to arrive at the best possible peptide, we performed
    <b>multiple iterations</b> of the engineering cycle. Each iteration involved designing our peptide, building it and
    finally testing it <i>in-silico</i>. The changes involved in each iteration and the information learnt have been
    documented.
  </p>

  <img src="https://static.igem.wiki/teams/4200/wiki/engineeringsuccess/engineering-success/dbtlcycle.png"
    alt="The design, build test learn cycle graphic" class="reveal-on-scroll">

  <h2 class="reveal-on-scroll"> The DBTL cycle <br> <small class="text-muted"><em>Design, Build, Test and
        Learn</em></small></h2>

  <p class="reveal-on-scroll"> The predicted structures of MAM7 and Heparin Binding Domain (HBD) of Fibronectin were
    docked with each other on ClusPro [1]-[4], followed by visualisation and observation of these interacting residues
    on PyMOL [5]. The sequence signature of the structural repeats in the proteins of Fibronectin and MAM7 were detected
    using HHrepID [6],[7]—a tool for <i>de novo</i> identification of repeats in protein sequences. This confirmed the
    existence of 5 continuous repeats of type 1 modules, forming the <b>Heparin-Binding Domain</b> of Fibronectin and
    the presence of seven Mammalian Cell Entry domains (MCEs) in MAM7. Fibronectin is known to require <b>five tandem
      MCE domains</b> to dock with MAM7; as a result, designing a peptide that would bind to one of the central MCE
    domains was deemed sufficient [8].
  </p>

  <p class="reveal-on-scroll">Each of the five repeats of the type I module present in Fibronectin was considered an
    individual peptide, and the structure for each of these peptides was predicted. Amber, an energy mininmization tool,
    was employed to stabilize the predicted peptide structures. The peptides were then docked with MAM7 to test its
    binding using ClusPro [1]-[4]. It was observed that the peptides numbered three and five derived from modules three
    and five of Fibronectin, respectively, docked with the MCE3 of MAM7, gave the highest docking scores.
  </p>

  <p class="reveal-on-scroll"> It was seen that the <b>fifth peptide</b> had the highest docking score with MCE3. Thus,
    this peptide was taken as the basis for the <b>next set of modifications</b>.
  </p>

  <figure class="figure reveal-on-scroll"> <img src="https://static.igem.wiki/teams/4200/wiki/multiplepeptide5.png"
      class="figure-img img-fluid rounded" alt="Fibronectin docked with MAM7">
    <figcaption class="figure-caption text-center"> Fig 1 :Fibronectin docked with MAM7(viewed on PyMOL[1]-[4])
    </figcaption>
  </figure>

  <p class="reveal-on-scroll"><b>Schrodinger Maestro</b>[9] was used to visualise and study the specific amino acids
    involved in the interaction between the peptide and MAM7. The residues within 5 Å were assumed to be close enough
    for the interaction and thus were considered during the evaluation. Further, this was verified by subjecting the
    peptide to an alanine scan on BUDE. An alanine scan replaces all the amino acid residues in a peptide with alanine
    one at a time and then compares the values of the respective Gibbs free energies to determine the extent of
    contribution of every amino acid in the docking process [10]. <b>Five</b> such mutatable positions were identified
    in the wild type. This was followed by improving the interactions by substituting aforementioned amino acids. These
    replacements were made based on the study [11], but each of them had a plethora of possibilities. For example,
    Methionine (present at the 29th position) could be replaced by Glutamic acid, Aspartic acid, Phenylalanine,
    Tyrosine, Tryptophan, Histidine or Leucine.
  </p>

  <p class="reveal-on-scroll">Therefore, the docked structure was subjected to <b>mutation Cutoff Scanning Matrix</b>
    (mCSM) [12], which helped us analyse which mutations were truly beneficial. With newly <b>learnt</b> information,
    the peptide was subjected to <b>new changes</b>. Hence, it was necessary to predict the structure of the peptide and
    dock it with MAM7 at every stage. The docked complex was subjected to an alanine scan to account for conformational
    changes by the substitutions, that mCSM does not account for, and to verify the interactions. This gave us twelve
    variants of the peptide with different combinations of the various substitutions that could be made. These were then
    visualised on Schrodinger Maestro and further analysed using <b>Prodigy</b> to identify the strongly interacting
    peptides.
  </p>

  <p class="reveal-on-scroll"><b>Four</b> of the <b>twelve</b> peptides showed an increased binding affinity, and the
    best among these was selected for the next round. The mutations performed in the previous iteration introduced new
    amino acids in the docked structure, opening up an avenue for a new set of substitutions and hence the process was
    repeated. Subsequently, a new set of variants were obtained (16 variants). These were further subjected to
    <b>docking</b> and analysis to select the next best peptide. At this stage, to prevent the peptides from mutually
    interacting and consequently decreasing their interaction efficiency with MAM7, we performed <b>AGGRESCAN</b> [13]
    on the four peptides obtained from the last iteration, to highlight the <b>hotspots of aggregation</b>. One such
    hotspot involving <b>eight</b> amino acids was observed and was rectified by substituting <b>Cytosine</b> at the
    34<sup>th</sup> position with <b>Asparagine</b> in every peptide. After this process, our very own
    software—<b>GRASP</b>, helped us increase the scope of possible mutations.
  </p>

  <p class="reveal-on-scroll">Two rounds of running the peptide through the software gave us the final one. The
    structure of the peptide was then predicted and docked with MAM7, and run through Prodigy [14],[15] to compare its
    binding affinity and K<sub>d</sub> value with the previous variants. These values are tabulated below:
  </p>

  <b>Table- Peptide variants with gibbs free energy and dissociation constant</b>

  <img src="https://static.igem.wiki/teams/4200/wiki/engineeringsuccess/ddg-table-finally.png"
    alt="Table file with peptide variant vs energy" class="reveal-on-scroll">
  <!--gap between the reference and table-->

  <b class="reveal-on-scroll">References:</b>

  <ol class="reveal-on-scroll">
    <li> I. T. Desta, K. A. Porter, B. Xia, D. Kozakov, and S. Vajda, “Performance and Its Limits in Rigid Body
      Protein-Protein Docking,” Structure, vol. 28, no. 9, pp. 1071-1081.e3, Sep. 2020, doi: 10.1016/j.str.2020.06.006.
    </li>
    <li>S. Vajda et al., “New Additions to the ClusPro Server Motivated by CAPRI”, doi: 10.1002/prot.25219. </li>
    <li>D. Kozakov et al., “The ClusPro web server for protein-protein docking”, doi: 10.1038/nprot.2016.169. </li>
    <li>D. Kozakov et al., “How Good is Automated Protein Docking?”, doi: 10.1002/prot.24403. </li>
    <li>PyMOL The PyMOL Molecular Graphics System, Version 2.5.2 Schrödinger, LLC. </li>
    <li> L. Zimmermann et al., “A Completely Reimplemented MPI Bioinformatics Toolkit with a New HHpred Server at its
      Core,” J. Mol. Biol., vol. 430, no. 15, pp. 2237–2243, Jul. 2018, doi: 10.1016/j.jmb.2017.12.007. </li>
    <li> F. Gabler et al., “Protein Sequence Analysis Using the MPI Bioinformatics Toolkit,” Curr. Protoc. Bioinforma.,
      vol. 72, no. 1, Dec. 2020, doi: 10.1002/cpbi.108. </li>
    <li>A. M. Krachler and K. Orth, “Functional characterization of the interaction between bacterial adhesin
      Multivalent Adhesion Molecule 7 (MAM7) protein and its host cell ligands,” J. Biol. Chem., vol. 286, no. 45, pp.
      38939–38947, Nov. 2011, doi: 10.1074/JBC.M111.291377. </li>
    <li>Maestro Schrödinger Release 2022-3:Maestro, Schrödinger, LLC, New York, NY, 2021. </li>
    <li>A. A. Ibarra et al., “Predicting and Experimentally Validating Hot-Spot Residues at Protein-Protein Interfaces,”
      ACS Chem. Biol., vol. 14, no. 10, pp. 2252–2263, Oct. 2019, doi:
      10.1021/ACSCHEMBIO.9B00560/SUPPL_FILE/CB9B00560_SI_002.XLSX. </li>
    <li> N. Z. Xie, Q. S. Du, J. X. Li, and R. B. Huang, “Exploring Strong Interactions in Proteins with Quantum
      Chemistry and Examples of Their Applications in Drug Design,” PLoS One, vol. 10, no. 9, p. 137113, Sep. 2015, doi:
      10.1371/JOURNAL.PONE.0137113. </li>
    <li>D. E. V. Pires, D. B. Ascher, and T. L. Blundell, “mCSM: predicting the effects of mutations in proteins using
      graph-based signatures,” Bioinformatics, vol. 30, no. 3, pp. 335–342, Feb. 2014, doi:
      10.1093/BIOINFORMATICS/BTT691. </li>
    <li>O. Conchillo-Solé, N. S. de Groot, F. X. Avilés, J. Vendrell, X. Daura, and S. Ventura, “AGGRESCAN: A server for
      the prediction and evaluation of ‘hot spots’ of aggregation in polypeptides,” BMC Bioinformatics, vol. 8, no. 1,
      pp. 1–17, Feb. 2007, doi: 10.1186/1471-2105-8-65/FIGURES/1. </li>
    <li>A. Vangone and A. M. Bonvin, “Contacts-based prediction of binding affinity in protein–protein complexes,”
      Elife, vol. 4, Jul. 2015, doi: 10.7554/eLife.07454. </li>
    <li> L. C. Xue, J. P. Rodrigues, P. L. Kastritis, A. M. Bonvin, and A. Vangone, “PRODIGY: a web server for
      predicting the binding affinity of protein–protein complexes,” Bioinformatics, p. btw514, Aug. 2016, doi:
      10.1093/bioinformatics/btw514. </li>
  </ol>

</div>

{% endblock %}