<!doctype html>
<html lang="en">

<head>
  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="https://static.igem.wiki/teams/4200/wiki/wiki/android-chrome-512x512.png" />
  <link rel="license" href="https://creativecommons.org/licenses/by/4.0/" />

  <!-- Bootstrap CSS -->
  <link href="/static/bootstrap.min.css" rel="stylesheet">

  <!-- Custom CSS -->
  <link href="/static/style.css" rel="stylesheet">

  <title>Software | MIT_MAHE - iGEM 2022</title>
</head>

<body>
  <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
  <div class="container">

    <!---- TEAM NAME ---->
    <a class="navbar-brand" href="./">MIT_MAHE</a>

    <!---- SMALL SCREEN MENU ICON ---->
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
      aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto left-aligned">

        <!---- HOME ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown"
            aria-expanded="false">
            Home
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="/">Home</a></li>
            <li><a class="dropdown-item" href="/medals/">Medals</a></li>
          </ul>
        </li>

        <!---- TEAM ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="/team/" id="navbarDropdown"
            role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Team
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="/team/">Team</a></li>
            <li><a class="dropdown-item" href="/attributions/">Attributions</a></li>
            <li><a class="dropdown-item" href="/collaborations/">Collaborations</a></li>
          </ul>
        </li>

        <!---- PROJECT ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown"
            aria-expanded="false">
            Project
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="/communication/">Communication</a></li>
            <li><a class="dropdown-item" href="/contribution/">Contribution</a></li>
            <li><a class="dropdown-item" href="/description/">Description</a></li>
            <li><a class="dropdown-item" href="/design/">Design</a></li>
            <li><a class="dropdown-item" href="/education/">Education</a></li>
            <li><a class="dropdown-item" href="/engineering/">Engineering</a></li>
            <li><a class="dropdown-item" href="/experiments/">Experiments</a></li>
            <li><a class="dropdown-item" href="/implementation/">Implementation</a></li>
            <li><a class="dropdown-item" href="/model/">Model</a></li>
            <li><a class="dropdown-item" href="/partnership/">Partnership</a></li>
            <li><a class="dropdown-item" href="/software/">Software</a></li>
          </ul>
        </li>

        <!---- HUMAN PRACTICES ---->
        <li class="nav-item">
          <a class="nav-link" href="/human-practices/">Human Practices</a>
        </li>

        <!---- AWARDS ---->
        <li class="nav-item dropdown">
          <a class="nav-link" href="/awards/">Awards</a>
        </li>

      </ul>
    </div>
  </div>
</nav>

  <!-- Header -->
  
  <header class="bg-hero py-5 mb-5">
    <div class="container.fluid h-100 reveal-on-scroll">
      <div class="row h-100 align-items-center">
        <div class="col-lg-12">
          <h1 class="display-2 mt-5 mb-2 page-header-text">Software</h1>
          <p class="lead mb-5 page-header-text-lead"></p>
        </div>
      </div>
    </div>
  </header>
  

  <!-- Page Content -->
  <div class="container">
    

<div class="row mt-4">
  <h2 class="reveal-on-scroll">Introduction</h2>
  <p class="reveal-on-scroll"><b>GRASP</b> - Generating random alanine scanned peptides is a peptide mutator that
    performs conserved substitutions based on inputs from an <b>alanine scan</b>. This was a crucial step in
    significantly increasing the scope of possible mutations. Existing softwares in the domain only consider
    surface-level information, ignoring any conformational changes that may occur. GRASP creates combinations of
    <b>conserved substitutions</b> on selected amino acids that fall within the specified range. Then, five random
    sequences are chosen from the sample set created. These are then tested and subjected to further analysis. Thus, our
    software is a one-of-a-kind approach to automating and exponentially simplifying the process of <b>random</b>
    conserved mutations. This process can be employed to potentially <b>increase the affinity</b> of the peptide to its
    interacting ligands.
  </p>

  <h1 class="reveal-on-scroll">GRASP</h1>
  <blockquote>
    <p class="reveal-on-scroll"><em><strong><span>Peptide sequence mutator through Alanine Scanning</span></strong></em>
    </p>
  </blockquote>

  <h3 class="reveal-on-scroll">About</h3>
  <blockquote>
    <p class="reveal-on-scroll"><em><span>For more information, refer to this page on our wiki: </span><strong> <a
            href="/engineering/" class='url'>Engineering success</a></strong></em><br>
      <em><span>Checkout the software at: </span><strong> <a href="https://gitlab.igem.org/2022/software-tools/mit-mahe"
            class='url'>MIT_MAHE Software tools</a></strong></em>
    </p>
  </blockquote>

  <figure class="figure reveal-on-scroll"> <img
      src="https://static.igem.wiki/teams/4200/wiki/software/cartesianproduct-manimce-v0-16-0-post0.gif"
      class="figure-img img-fluid rounded" alt="GRASP GIF animation" style="width: 85%;">
    <figcaption class="figure-caption text-center"> Peptide sequence mutations on a sample sequence. Right: The list of
      all the output sequences. </figcaption>
  </figure>

  <h3 class="reveal-on-scroll">Description</h3>
  <p class="reveal-on-scroll"><span>The software comprises of the following modules:</span></p>
  <ol class="reveal-on-scroll">
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Alanine Scan \( \Delta \Delta G \) values</span></strong> <span>BUDE
          Alanine scan is employed to obtain \( \Delta \Delta G \) values of each residue present in the input.</span>
      </p>
    </li>
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Conservative mutations</span></strong></p>
      <p class="reveal-on-scroll"><span>Conserved mutation is a process in which mutations are made within the same type
          of residues.</span> </p>
      <p class="reveal-on-scroll"><em><span>For example:</em><span> We replaced amino acids within the same group, for
          example, hydrophobic re 3—hydrophobic residues with hydrophobic residues or amino acids of same charge with
          each other.</span></p>
    </li>
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Random sampler</span></strong></p>
      <p class="reveal-on-scroll"><span>Random peptides are selected as samples from the huge sequence list received to
          reduce the user workload. Note that this selection is <b>pseudo-random</b>.</span> </p>
    </li>
  </ol>

  <h3 class="reveal-on-scroll"><span>Features</span></h3>

  <figure class="figure reveal-on-scroll"> <img src="https://static.igem.wiki/teams/4200/wiki/software/grasp-help.png"
      class="figure-img img-fluid rounded" alt="GRASP Help" width="80%">
    <figcaption class="figure-caption text-center"> GRASP Help </figcaption>
  </figure>

  <ol class="reveal-on-scroll">
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Command Line Interface (CLI)</span></strong> <span>We can use the
          software locally on command prompt.</span> </p>
    </li>
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Integrable modules</span></strong></p>
      <p class="reveal-on-scroll"><span>Our software consists of different modules for the mutator and alanine scan to
          integrate them into different and more extensive networks by other iGEM teams and parties.</span></p>
    </li>
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Mutation lock</span></strong> <span>To prevent the mutation of an
          already modified residue, that part of the sequence was put under a mutation lock</span> </p>
    </li>
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll"><strong><span>Aggrescan</span></strong></p>
      <p class="reveal-on-scroll"><span>Inclusion of a <b>BASH</b> script, which takes the output and prepends FASTA
          headers to each sequence in the text file.</span></p>
    </li>
  </ol>

  <h3 class="reveal-on-scroll"><span>Background</span></h3>
  <p class="reveal-on-scroll"><span>Protein engineering is modifying a protein through substitution, insertion, or
      deletion to exhibit desired characteristics and properties.</span></p>
  <p class="reveal-on-scroll"><span>During the process of designing our peptide, we came across the problem of having an
      <b>extensive sample set</b> of substitutions through trial and error. However, even after this, we could not
      guarantee an increase in affinity compared to the original peptide as needed in our case. To overcome this
      problem, we designed this software to provide all the most probable mutations that increase the binding affinity.
    </span></p>
  <p class="reveal-on-scroll"><span>Furthermore, we explored other point mutation softwares for our problem; however,
      they only took surface information into consideration. We decided to take \( \Delta \Delta G \) value of a residue
      as the parameter for mutation which give an increase in the accuracy of the mutation.</span></p>

  <h2 class="reveal-on-scroll">Method</h2>

  <h3 class="reveal-on-scroll">\( \Delta \Delta G \) threshold</h3>
  <p class="reveal-on-scroll"> We take \( \Delta \Delta G \)value of each residue derived from BUDE alanine scan as a
    parameter to do mutations. To make these mutations we <b>set a threshold</b> \( \Delta \Delta G \)range for each
    residue (in our case, 0 to 1). Only if the \( \Delta \Delta G \) value lies between this range will it be mutated.
  </p>

  <h3 class="reveal-on-scroll">Significance of Cartesian product</h3>
  <p class="reveal-on-scroll"> Cartesian Product: The Cartesian Product of sets \(A\) and \(B\) is defined as the set of
    all ordered pairs \((x, y)\) such that \(x\) belongs to \(A\)and \(y\) belongs to \(B\) For example, if \(A = {1,
    2}\) and \(B = {3, 4, 5}\), then the Cartesian Product of A and B is \({(1, 3), (1, 4), (1, 5), (2, 3), (2, 4), (2,
    5)}\). Originally, during a test run, the computational time that the software took for making 3 mutation positions
    was 2 hours 20 mins. After incorporating the cartesian product into the mutator, the time reduced to 11 mutation
    positions in 10 seconds.
  </p>

  <h3 class="reveal-on-scroll">Algorithm</h3>
  <p class="reveal-on-scroll"> We first create a Mutater object, which support the following three formats:
  </p>
  <ol>
    <li class="reveal-on-scroll"> Wild type amino acid, the position of the Amino acid, and the Mutant type it needs to
      be replaced with. </li>
    <li class="reveal-on-scroll"> Just the Wild type amino acid and its position, without the Mutant type. This can act
      as the template to create
      new Mutater objects. </li>
    <li class="reveal-on-scroll"> Just the position in the sequence. </li>
  </ol>

  <h3 class="reveal-on-scroll">Alanine scanning</h3>
  <p class="reveal-on-scroll"> The Alanine scanning results from BUDE Alanine scan are parsed through which results in a
    position array over which mutations are possibly required.
  </p>

  <h3 class="reveal-on-scroll">Group mutations</h3>
  <p class="reveal-on-scroll"> The sequence is taken as a template over which a special array is constructed, with the
    following steps:
  </p>
  <ol class="reveal-on-scroll">
    <li class="reveal-on-scroll"> Iterate over the sequence and get the wild type amino acid at each stage. </li>
    <li class="reveal-on-scroll"> Retain the wild type if mutation isn't required at this position. This could result
      when the position doesn't
      want to be mutated, or when the position is present in the mutation lock array. </li>
    <li class="reveal-on-scroll"> If a mutation is required in the current position, replace the position with an array
      of all the possible amino
      acids within the same group. This step is important to achieve Conserved mutations. </li>
  </ol>

  <h3 class="reveal-on-scroll">Cartesian product</h3>
  <p class="reveal-on-scroll"> The special array is then passed to the <code>itertools.product()</code> function, which
    evaluates the Cartesian product on the elements of the special array. This results in an iterable object that
    contains all the final sequences, which is then stored in the text file.
  </p>

  <h3 class="reveal-on-scroll">Random sample</h3>
  <p class="reveal-on-scroll"> A trivial random sampler is implemented using <code>random.sample()</code> on the
    sequences array, and then stored in a text file.
  </p>

</div>


  </div>

  <!-- Footer: MUST mention license AND have a link to team wiki's repository on gitlab.igem.org -->
  <footer class="pt-5 pb-5 footer py-5 mt-5 text-white">
  <div class="container">
    <div class="row mb-4 footer-parent">
      <div class="col-lg-6 col-xs-12">
        <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/footer-logo.png" alt="AMPIFIN-footer-logo">
        <p class="h5 footer-logo-text"><em>A sustainable solution for Vibriosis in fish</em></p>
      </div>
      <div class="col-1 footer-socials">
        <a href="mailto:manipalbiomachines.igem@gmail.com">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/envelope.svg" alt="mail" width="32"
            height="32">
        </a>
      </div>
      <div class="col-1 footer-socials">
        <a href="https://www.linkedin.com/company/manipal-biomachines/">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/linkedin.svg" alt="linkedin" width="32"
            height="32">
        </a>
      </div>
      <div class="col-1 footer-socials">
        <a href="https://instagram.com/manipalbiomachines?igshid=YmMyMTA2M2Y=">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/instagram.svg" alt="instagram" width="32"
            height="32">
        </a>
      </div>
      <div class="col-1 footer-socials">
        <a href="https://twitter.com/m_biomachines?t=EmCfYt5V-Q6pa5j1KyXSYA&s=09">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/twitter.svg" alt="twitter" width="32"
            height="32">
        </a>
      </div>
    </div>
    <hr>
    <!-- The following MUST be on every page: license information and link to the repository on gitlab.igem.org -->
    <div class="row mt-4">
      <div class="col">
        <p class="mb-0"><small>© 2022 - Content on this site is licensed under a <a class="subfoot"
              href="https://creativecommons.org/licenses/by/4.0/" rel="license">Creative Commons Attribution 4.0
              International license</a>.</small></p>
        <p><small>The repository used to create this website is available at <a
              href="https://gitlab.igem.org/2022/mit-mahe">gitlab.igem.org/2022/mit-mahe</a>.</small></p>
      </div>
    </div>
  </div>
</footer>

  <!-- Wiki Tools: Teams are allowed to remove it -->
  <!-- <style>
    .wiki-tools-footer {
        --bs-gutter-x: 1.5rem;
        background: #222124;
        margin: 0;
        max-width: 100%;
        padding: 1rem 3rem;
        display: flex;
        gap: 1.5em;
    }

    .wiki-tools-footer a {
        text-decoration: none;
        color: white;
        font-weight: 400;
        transition: color 0.3s;
    }

    .wiki-tools-footer a:hover {
        color: #08C784;
    }

    .wiki-tools-footer img {
        object-fit: contain;
    }
</style>

<section class="wiki-tools-footer container">
    <a id="wiki-tools-link">Access Wiki Tools</a>
    <a id="wiki-tools-repo">View Source Code</a>
    <img id="wiki-tools-pipeline">
</section>

<script>
    const wikiToolsLink = document.getElementById('wiki-tools-link')
    const wikiToolsRepo = document.getElementById('wiki-tools-repo')
    const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')

    const path = document.location.pathname.split('/')

    const competitionSlug = document.location.host.split('.')[0]
    const teamSlug = path[1]
    const page = path[2] === '' ? 'index' : path[2]

    wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
    wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
    wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
</script> -->

  <!-- Bootstrap Bundle with Popper -->
  <script src="/static/bootstrap.bundle.min.js"></script>

  <script defer src="/static/script.js"></script>
  <script src="/static/jquery-3.6.1.min.js"></script>
  <script src="/static/masonry.pkgd.min.js"></script>
  <script src="/static/imagesloaded.pkgd.min.js"></script>
  <script src="/static/mathjax.js"></script>
</body>

</html>