<!doctype html>
<html lang="en">

<head>
  <!-- Required meta tags -->
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <link rel="shortcut icon" href="https://static.igem.wiki/teams/4200/wiki/wiki/android-chrome-512x512.png" />
  <link rel="license" href="https://creativecommons.org/licenses/by/4.0/" />

  <!-- Bootstrap CSS -->
  <link href="/2022-igem-wiki/static/bootstrap.min.css" rel="stylesheet">

  <!-- Custom CSS -->
  <link href="/2022-igem-wiki/static/style.css" rel="stylesheet">

  <title>Model | MIT_MAHE - iGEM 2022</title>
</head>

<body>
  <!-- Navigation -->
  <nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top">
  <div class="container">

    <!---- TEAM NAME ---->
    <a class="navbar-brand" href="./">MIT_MAHE</a>

    <!---- SMALL SCREEN MENU ICON ---->
    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent"
      aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
      <span class="navbar-toggler-icon"></span>
    </button>

    <div class="collapse navbar-collapse" id="navbarSupportedContent">
      <ul class="navbar-nav ml-auto left-aligned">

        <!---- HOME ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown"
            aria-expanded="false">
            Home
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="/2022-igem-wiki/">Home</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/medals/">Medals</a></li>
          </ul>
        </li>

        <!---- TEAM ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="/2022-igem-wiki/team/" id="navbarDropdown"
            role="button" data-bs-toggle="dropdown" aria-expanded="false">
            Team
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="/2022-igem-wiki/team/">Team</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/attributions/">Attributions</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/collaborations/">Collaborations</a></li>
          </ul>
        </li>

        <!---- PROJECT ---->
        <li class="nav-item dropdown">
          <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-bs-toggle="dropdown"
            aria-expanded="false">
            Project
          </a>
          <ul class="dropdown-menu" aria-labelledby="navbarDropdown">
            <li><a class="dropdown-item" href="/2022-igem-wiki/communication/">Communication</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/contribution/">Contribution</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/description/">Description</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/design/">Design</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/education/">Education</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/engineering/">Engineering</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/experiments/">Experiments</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/implementation/">Implementation</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/model/">Model</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/partnership/">Partnership</a></li>
            <li><a class="dropdown-item" href="/2022-igem-wiki/software/">Software</a></li>
          </ul>
        </li>

        <!---- HUMAN PRACTICES ---->
        <li class="nav-item">
          <a class="nav-link" href="/2022-igem-wiki/human-practices/">Human Practices</a>
        </li>

        <!---- AWARDS ---->
        <li class="nav-item dropdown">
          <a class="nav-link" href="/2022-igem-wiki/awards/">Awards</a>
        </li>

      </ul>
    </div>
  </div>
</nav>

  <!-- Header -->
  
  <header class="bg-hero py-5 mb-5">
    <div class="container.fluid h-100 reveal-on-scroll">
      <div class="row h-100 align-items-center">
        <div class="col-lg-12">
          <h1 class="display-2 mt-5 mb-2 page-header-text">Model</h1>
          <p class="lead mb-5 page-header-text-lead"></p>
        </div>
      </div>
    </div>
  </header>
  

  <!-- Page Content -->
  <div class="container">
    

<div class="row">
  <h1 class="reveal-on-scroll">Molecular Dynamic Simulations</h1>
  <p class="reveal-on-scroll">
    Determining the behaviour of a protein or peptide in response to environmental stresses is crucial in
    establishing
    the efficacy of the peptide in a system. Thus, molecular dynamics simulation can help us subject the peptide to
    different conditions that mimic the natural environment that it may exist in, <i>in silico</i>.
  </p>
  <p class="reveal-on-scroll">
    Multiple tools can be used to perform MDS, such as GROMACS [1][2][3], CHARMM [4], AMBER [5] and Desmond [6].
    However, MDS requires high
    computational power. Given our limited resources, we partnered with IISER Pune II, to run molecular dynamics
    simulations on their supercomputer PARAM. The results were then analysed by us to check for the stability of the
    peptide in the given environment.
  </p>
  <p class="reveal-on-scroll">
    We discovered that a pH of 7.5 and a temperature of 298K are the typical characteristics of an aquaculture system
    for consumable fish through our human practices and research. These parameters were applied to a cubic system and
    an
    OPLS-AA/M force field was applied across the system to analyse the dynamic changes in the conformation of the
    peptide.
  </p>

  <h3 class="reveal-on-scroll">RMSD</h3>

  <figure class="figure reveal-on-scroll">
    <img src="https://static.igem.wiki/teams/4200/wiki/model/model-rmds.png" class="figure-img img-fluid rounded"
      alt="RMS v/s time" width="70%">
    <figcaption class="figure-caption text-center">
      Fig 1: Root Mean Square Distance (nm) versus Time (ps) from GROMACS
    </figcaption>
  </figure>


  <i></i>
  <p class="reveal-on-scroll">RMSD or root mean squared distance is the numerical representation of the difference
    between the peptide's
    initial
    structure and the structure at a given moment in time. Generally, an RMSD variation of less than 2 Angstorms is
    supposed to be very close and stable.
    \[ \text{RMSD} = \sqrt{\frac{\sum [m_i(x_i-y_i)^2]}{M}} \]
  </p>

  <p class="reveal-on-scroll">The RMSD value was seen to increase for the first 50ns, which is a given due to the
    peptide stabilizing to the
    given conditions in the system. It was observed that the RMSD values decreased briefly and then stabilized with a
    fluctuation of approximately 1 Angstorms, which falls comfortably below the accepted threshold of
    2 Angstorms, indicating that our peptide is very stable in the given environment. It also shows that <b>no
      conformational changes</b> occur within the first 100 ns of the simulation, which could otherwise hamper its
    target interaction.</p>

  <h3 class="reveal-on-scroll">RMSF</h3>

  <figure class="figure reveal-on-scroll">
    <img src="https://static.igem.wiki/teams/4200/wiki/model/model-rmsf.jpg" class="figure-img img-fluid rounded"
      alt="RMS v/s residue" width="70%">
    <figcaption class="figure-caption text-center">
      Fig 2: Root Mean Square Fluctuation (nm) versus Residue from GROMACS
    </figcaption>
  </figure>

  <p class="reveal-on-scroll">RMSF or root mean squared fluctuations is a calculation of individual residue flexibility,
    or how much a
    particular
    residue moves (fluctuates) during a simulation.
    \[
    \text{RMSF} = {\sqrt{\frac{\sum [m_i(x_i(t_j)-x_{ref})^2]}{T}}}
    \]
  </p>
  <p class="reveal-on-scroll">The residues at the very beginning and end of our peptide show the highest fluctuations
    and this indicates that
    they are less stable than the residues with fewer fluctuations. For example, the amino acid residues between 14 to
    24 are much more stable than the residues from 36 to 40, which correspond to loops in the structure of the
    peptide.
  </p>
  <h3 class="reveal-on-scroll">Radius of Gyration</h3>


  <figure class="figure reveal-on-scroll">
    <img src="https://static.igem.wiki/teams/4200/wiki/model/model-rg.jpg" class="figure-img img-fluid rounded"
      alt="Radius of gyration v/s time" width="70%">
    <figcaption class="figure-caption text-center">
      Fig 3: Radius of Gyration (nm) versus Time (ps) from GROMACS
    </figcaption>
  </figure>


  <p class="reveal-on-scroll">The radius of gyration for a given molecule is a measure of the compactness of the
    molecule or how far apart the
    mass of the object is distributed around its centre of mass.
    \[
    R_g = \sqrt{\frac{\sum [m_i(x_i-x_c)^2]}{M}}
    \]</p>

  <p class="reveal-on-scroll">From the resulting plot of the radius of gyration versus time, we can see that the net
    radius of gyration remains
    fairly constant and is <b>within the range of 1 Angstorm</b>. This indicates that the peptide does not fold
    or
    unfold
    over time, thereby contributing to no change in the compactness of the peptide. We can also conclude that the
    peptide does not undergo any major conformational changes which can result in changes in the folded volume of the
    protein, and thereby talks about the <b>stability of the peptide</b>.</p>
  <p class="reveal-on-scroll">MD Simulations on the MAM7-peptide dock would give a lot more information about the
    interactions present, the
    stability of the complex, interaction energies, compactness and the effect of conformational changes on the
    interaction. Parameters such as RMSF would give crucial information regarding the interacting residues present in
    the complex. Induced conformational changes upon docking could be analyzed using graphs for RMSD as well as
    observing the trajectory file. We planned on performing these simulations to analyze the MAM7-peptide interaction
    but were limited by time and resources.</p>

  <h1 class="reveal-on-scroll">IC<sub>50</sub></h1>
  <p class="reveal-on-scroll">The <b>optimum dosage values</b> of the peptide is essential for effective prevention of
    the disease Since both
    the
    peptide and fibronectin are in a position to interact with MAM7, we concluded that our peptide would act as a
    <b>competitive inhibitor</b> against fibronectin. Therefore, our mathematical model describes the relationship
    between the
    concentration of peptide required for a given concentration of fibronectin present in the system.
  </p>
  <ul class="reveal-on-scroll">
    <li class="reveal-on-scroll">IC<sub>50</sub> is the concentration of the drug or displacing ligand in a competitive inhibition mechanism
      that
      reduces or
      inhibits the receptor-ligand interaction by 50%. It is a measure of the potency of a drug.</li>
    <li class="reveal-on-scroll">IC<sub>50</sub> values are typically calculated via an IC<sub>50</sub> assay using a serial dilution of the
      drug. Based
      on the
      definition of IC<sub>50</sub>, we arrived at an equation for the IC<sub>50</sub> concentration for our
      antimicrobial
      peptide</li>

    <li class="reveal-on-scroll">Our peptide competitively binds to MAM7 and inhibits its interaction with Fibronectin, therefore fitting the
      reaction mechanism. We derived an equation for the IC<sub>50</sub> value of our peptide with the following
      steps:

      <ol class="reveal-on-scroll">
        <li class="reveal-on-scroll">
          <p class="reveal-on-scroll">
            The reaction mechanism can be considered to be a parallel reaction scheme where MAM7 gets partitioned
            between
            Fibronectin and our peptide.
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-1.png" alt="M+F=MF" width="30%"
              class="reveal-on-scroll">
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-2.png" alt="M+P=MP" width="30%"
              class="reveal-on-scroll">
          </p>
          <p class="reveal-on-scroll">
            Equations for the Association and Dissociation constant can be written as follows:
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-3-4.png"
              alt="Association and Dissociation Equations" width="30%" class="reveal-on-scroll">
          </p>
          <p class="reveal-on-scroll">
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-5-6.png"
              alt="Association and Dissociation Constants" width="30%" class="reveal-on-scroll">
          </p>
        </li>
        <li class="reveal-on-scroll">
          <p class="reveal-on-scroll">Given the total concentration of MAM7 in a system, the MAM7 would be partitioned
            between the
            MAM7-Fibronectin
            complex, MAM7-peptide complex and free MAM7.
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-7.png" alt="Total M concentration"
              width="30%" class="reveal-on-scroll">
          </p>
        </li>
        <li class="reveal-on-scroll">
          <p class="reveal-on-scroll">
            We defined \(\rho\) as the ratio of the concentration of the MAM7-Fibronectin complex to the total MAM7
            concentration.
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-8.png" alt="Rho derivation" , width="70%"
              class="reveal-on-scroll">
          </p>
        </li>
        <li class="reveal-on-scroll">
          <p class="reveal-on-scroll">
            We defined \(\rho_{in}\) as the interaction ratio in the absence of our peptide.
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-9.png" alt="Rho in derivation" width="30%"
              class="reveal-on-scroll">
          </p>
        </li>
        <li class="reveal-on-scroll">
          <p class="reveal-on-scroll">
            The below relation was arrived at from the definition of IC<sub>50</sub>.
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-10.png" alt="IC50 relation" width="50%"
              class="reveal-on-scroll">
          </p>
        </li>
        <li class="reveal-on-scroll">
          <p class="reveal-on-scroll">
            On simplification, the final equation for IC<sub>50</sub> value of our peptide was found.
            <img src="https://static.igem.wiki/teams/4200/wiki/model/model-11.png" alt="Final IC50 equation" width="30%"
              class="reveal-on-scroll">
          </p>
        </li>
      </ol>
    </li>
    <li class="reveal-on-scroll">
      <p class="reveal-on-scroll">
        The dissociation constant values substituted here were predicted using <b>PRODIGY</b>, thereby, giving us a
        prediction of
        the IC<sub>50</sub> concentration. The actual values of the dissociation constants would be found using
        Isothermal
        Calorimetric analysis.
      </p>
      <p class="reveal-on-scroll">The <b>PRODIGY</b> predicted values are:</p><br>
      <img src="https://static.igem.wiki/teams/4200/wiki/model/model-12.png"
        alt="Predicted dissociation constant values" width="20%" class="reveal-on-scroll">
    </li>
  </ul>

  <figure class="figure reveal-on-scroll">
    <img src="https://static.igem.wiki/teams/4200/wiki/model/model-ic50-2.png"
      class="figure-img img-fluid rounded reveal-on-scroll" alt="fibronectin concentrations" width="80%">
    <figcaption class="figure-caption text-center">
      Fig 4: \(IC_{50}\) values plotted for a range of Fibronectin concentrations
    </figcaption>
  </figure>

  <p class="reveal-on-scroll">As an example,
  <ul class="reveal-on-scroll">
    <li class="reveal-on-scroll">MW of peptide = 4 kDa = 4 kg/mol</li>
    <li class="reveal-on-scroll">Say [F] = 1 nM</li>
    <li class="reveal-on-scroll">IC<sub>50</sub> = 10.67 microM = 42 mg/L</li>
  </ul>
  </p>
  <b class="reveal-on-scroll">References</b>
  <ol class="reveal-on-scroll">
    <li class="reveal-on-scroll">
      S. Pronk et al., “GROMACS 4.5: a high-throughput and highly parallel open source molecular simulation toolkit,”
      Bioinformatics, vol. 29, no. 7, pp. 845–854, Apr. 2013, doi: 10.1093/BIOINFORMATICS/BTT055.
    </li>
    <li class="reveal-on-scroll">
      M. J. Abraham et al., “GROMACS: High performance molecular simulations through multi-level parallelism from
      laptops to supercomputers,” SoftwareX, vol. 1–2, pp. 19–25, Sep. 2015, doi: 10.1016/J.SOFTX.2015.06.001.
    </li>
    <li class="reveal-on-scroll">
      S. Páll, M. J. Abraham, C. Kutzner, B. Hess, and E. Lindahl, “Tackling exascale software challenges in molecular
      dynamics simulations with GROMACS,” Lect. Notes Comput. Sci. (including Subser. Lect. Notes Artif. Intell. Lect.
      Notes Bioinformatics), vol. 8759, pp. 3–27, 2015, doi: 10.1007/978-3-319-15976-8_1/FIGURES/7.
    </li>
    <li class="reveal-on-scroll">
      B. R. Brooks et al., “CHARMM: The Biomolecular Simulation Program,” J. Comput. Chem., vol. 30, no. 10, p. 1545,
      Jul. 2009, doi: 10.1002/JCC.21287.
    </li>
    <li class="reveal-on-scroll">
      D.A. Case, H.M. Aktulga, K. Belfon, I.Y. Ben-Shalom, J.T. Berryman, S.R. Brozell, D.S. Cerutti, T.E. Cheatham,
      III, G.A. Cisneros, V.W.D. Cruzeiro, T.A. Darden, R.E. Duke, G. Giambasu, M.K. Gilson, H. Gohlke, A.W. Goetz, R.
      Harris, S. Izadi, S.A. Izmailov, K. Kasavajhala, M.C. Kaymak, E. King, A. Kovalenko, T. Kurtzman, T.S. Lee, S.
      LeGrand, P. Li, C. Lin, J. Liu, T. Luchko, R. Luo, M. Machado, V. Man, M. Manathunga, K.M. Merz, Y. Miao, O.
      Mikhailovskii, G. Monard, H. Nguyen, K.A. O'Hearn, A. Onufriev, F. Pan, S. Pantano, R. Qi, A. Rahnamoun, D.R.
      Roe,
      A. Roitberg, C. Sagui, S. Schott-Verdugo, A. Shajan, J. Shen, C.L. Simmerling, N.R. Skrynnikov, J. Smith, J.
      Swails, R.C. Walker, J. Wang, J. Wang, H. Wei, R.M. Wolf, X. Wu, Y. Xiong, Y. Xue, D.M. York, S. Zhao, and P.A.
      Kollman (2022), Amber 2022, University of California, San Francisco.
    </li>
    <li class="reveal-on-scroll">Kevin J. Bowers, Edmond Chow, Huafeng Xu, Ron O. Dror, Michael P. Eastwood, Brent A. Gregersen, John L.
      Klepeis,
      Istvan Kolossvary, Mark A. Moraes, Federico D. Sacerdoti, John K. Salmon, Yibing Shan, and David E. Shaw,
      "Scalable Algorithms for Molecular Dynamics Simulations on Commodity Clusters," Proceedings of the ACM/IEEE
      Conference on Supercomputing (SC06), Tampa, Florida, 2006, November 11-17

      <p class="reveal-on-scroll">Schrödinger Release 2022-3: Desmond Molecular Dynamics System, D. E. Shaw Research,
        New York, NY, 2021.
        Maestro-Desmond Interoperability Tools, Schrödinger, New York, NY, 2021.</p>
    </li>
  </ol>

</div>


  </div>

  <!-- Footer: MUST mention license AND have a link to team wiki's repository on gitlab.igem.org -->
  <footer class="pt-5 pb-5 footer py-5 mt-5 text-white">
  <div class="container">
    <div class="row mb-4 footer-parent">
      <div class="col-lg-6 col-xs-12">
        <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/footer-logo.png" alt="AMPIFIN-footer-logo">
        <p class="h5 footer-logo-text"><em>A sustainable solution for Vibriosis in fish</em></p>
      </div>
      <div class="col-1 footer-socials">
        <a href="mailto:manipalbiomachines.igem@gmail.com">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/envelope.svg" alt="mail" width="32"
            height="32">
        </a>
      </div>
      <div class="col-1 footer-socials">
        <a href="https://www.linkedin.com/company/manipal-biomachines/">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/linkedin.svg" alt="linkedin" width="32"
            height="32">
        </a>
      </div>
      <div class="col-1 footer-socials">
        <a href="https://instagram.com/manipalbiomachines?igshid=YmMyMTA2M2Y=">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/instagram.svg" alt="instagram" width="32"
            height="32">
        </a>
      </div>
      <div class="col-1 footer-socials">
        <a href="https://twitter.com/m_biomachines?t=EmCfYt5V-Q6pa5j1KyXSYA&s=09">
          <img src="https://static.igem.wiki/teams/4200/wiki/wiki/footer/twitter.svg" alt="twitter" width="32"
            height="32">
        </a>
      </div>
    </div>
    <hr>
    <!-- The following MUST be on every page: license information and link to the repository on gitlab.igem.org -->
    <div class="row mt-4">
      <div class="col">
        <p class="mb-0"><small>© 2022 - Content on this site is licensed under a <a class="subfoot"
              href="https://creativecommons.org/licenses/by/4.0/" rel="license">Creative Commons Attribution 4.0
              International license</a>.</small></p>
        <p><small>The repository used to create this website is available at <a
              href="https://gitlab.igem.org/2022/mit-mahe">gitlab.igem.org/2022/mit-mahe</a>.</small></p>
      </div>
    </div>
  </div>
</footer>

  <!-- Wiki Tools: Teams are allowed to remove it -->
  <!-- <style>
    .wiki-tools-footer {
        --bs-gutter-x: 1.5rem;
        background: #222124;
        margin: 0;
        max-width: 100%;
        padding: 1rem 3rem;
        display: flex;
        gap: 1.5em;
    }

    .wiki-tools-footer a {
        text-decoration: none;
        color: white;
        font-weight: 400;
        transition: color 0.3s;
    }

    .wiki-tools-footer a:hover {
        color: #08C784;
    }

    .wiki-tools-footer img {
        object-fit: contain;
    }
</style>

<section class="wiki-tools-footer container">
    <a id="wiki-tools-link">Access Wiki Tools</a>
    <a id="wiki-tools-repo">View Source Code</a>
    <img id="wiki-tools-pipeline">
</section>

<script>
    const wikiToolsLink = document.getElementById('wiki-tools-link')
    const wikiToolsRepo = document.getElementById('wiki-tools-repo')
    const wikiToolsPipeline = document.getElementById('wiki-tools-pipeline')

    const path = document.location.pathname.split('/')

    const competitionSlug = document.location.host.split('.')[0]
    const teamSlug = path[1]
    const page = path[2] === '' ? 'index' : path[2]

    wikiToolsLink.href = `https://tools.igem.org/wiki-tools/${competitionSlug}/${teamSlug}/${page}`
    wikiToolsRepo.href = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}`
    wikiToolsPipeline.src = `https://gitlab.igem.org/${competitionSlug}/${teamSlug}/badges/main/pipeline.svg`
</script> -->

  <!-- Bootstrap Bundle with Popper -->
  <script src="/2022-igem-wiki/static/bootstrap.bundle.min.js"></script>

  <script defer src="/2022-igem-wiki/static/script.js"></script>
  <script src="/2022-igem-wiki/static/jquery-3.6.1.min.js"></script>
  <script src="/2022-igem-wiki/static/masonry.pkgd.min.js"></script>
  <script src="/2022-igem-wiki/static/imagesloaded.pkgd.min.js"></script>
  <script src="/2022-igem-wiki/static/mathjax.js"></script>
</body>

</html>